spec/services/questions_service_spec.rb

```diff
@@ -0,0 +1,179 @@
+require "rails_helper"
+
+RSpec.describe QuestionsService do
+  subject(:question_service) { described_class.call(question_params) }
+
+  let(:question_params) do
+    {
+      request_id:,
+      proceedings:,
+    }.to_json
+  end
+  let(:request_id) { SecureRandom.uuid }
+  let(:proceedings) do
+    [
+      {
+        ccms_code:,
+        delegated_functions_used: true,
+        client_involvement_type:,
+      },
+    ]
+  end
+  let(:ccms_code) { "DA005" }
+  let(:client_involvement_type) { "A" }
+
+  context "when the request succeeds" do
+    context "when there is a DA proceeding with CIT of applicant" do
+      it "returns valid response with expected tasks" do
+        expect(question_service).to eq expected_da005_applicant_response
+      end
+    end
+
+    context "when there is a DA proceeding with CIT of Defendant" do
+      let(:client_involvement_type) { "D" } # expect Q7&8 to appear
+
+      it "returns valid response with expected tasks" do
+        expect(question_service).to eq expected_da005_defendant_response
+      end
+    end
+
+    context "when there is a DA proceeding as Applicant and a Child Section 8 with CIT of Defendant" do # expect Q7&8 not to appear
+      let(:proceedings) do
+        [
+          {
+            ccms_code: "DA005",
+            delegated_functions_used: true,
+            client_involvement_type: "A",
+          },
+          {
+            ccms_code: "SE013",
+            delegated_functions_used: true,
+            client_involvement_type: "D",
+          },
+        ]
+      end
+
+      it "returns valid response with expected tasks" do
```

Comment:
Can we make this something similar to "it returns valid response without questions 7 and 8" or "it returns valid response without domestic abuse/non applicant questions" and remove comment above?